### ISSUE TITLE
Support isolate view options

### ISSUE THREAD

**Issue by BigRoy** [abstractfactory/maya-capture#61]
# Problem

Currently there's no way to parse and/or apply any Isolate View options to capture only a subset of what is in the scene.

# Solution

Add the options related to the `modelEditor` in the viewport options or amongst a new Options dictionary. And also implement the "parsing" of the settings from a view/panel.

**Comment by mottosso** [comment]
Are you referring to this option?

> [isolate (list)](https://github.com/abstractfactory/maya-capture/blob/86a98f0aaf5a23ca79322356378fe997eaf597e9/capture.py#L64): List of nodes to isolate upon capturing

And that it should be included in the results of `capture.parse_view()`, and applied via `capture.apply_view()`?

**Comment by BigRoy** [comment]
Ha, yes. Exactly that option actually! 

I remembered seeing it in the docstring so quickly glanced over it, but apparently missed it this afternoon, twice. Sorry about that and thanks fo pointing me to it.

Aside from that particular option I'm in doubt of having it parse with the `parse_view` and `apply_view` because it's so related to a specific state of the scene (the nodes actually existing), as such less stateless than the other options. For example it wouldn't be as interesting in a preset of sorts.

What are your thoughts/ideas?

**Comment by mottosso** [comment]
> For example it wouldn't be as interesting in a preset of sorts.

That's a good point. With that, I think we should leave it out.

**Comment by BigRoy** [comment]
I was thinking maybe just adding an example to the readme of how to retrieve the currently active objects and pass it to `capture()` and leave it at that.

That way we've handled the use case for those who are looking for it.

**Comment by mottosso** [comment]
Yeah, that could work. Will you have a look at that?

**Comment by BigRoy** [comment]
Sure will. Probably tomorrow!

**Comment by mottosso** [comment]
Thanks Roy!

**Comment by BigRoy** [comment]
An example is added in https://github.com/BigRoy/maya-capture/commit/611a08c7b83129b512967fcd61a693800dcbc8f4

Notice that you'll need the previous commit https://github.com/BigRoy/maya-capture/commit/8f9ba4a5ff0b5f76d1f38a26e0cb5f1e76438509 as well to have the `parse_active_panel()` function in capture. This simplified things a lot which I felt was good.

What do you think?

**Comment by mottosso** [comment]
Looks good. Could you make it a PR? Then I can comment on individual lines.

**Comment by BigRoy** [comment]
The original proposal here hasn't been implemented, yet it *has been documented* as was proposed in the discussion. 

This documentation was implemented in the README with PR #62

**Comment by mottosso** [comment]
> The original proposal here hasn't been implemented

Hadn't it already been implemented? Or which proposal are you referring to?

**Comment by BigRoy** [comment]
> Hadn't it already been implemented? 

Ah, actually you're correct. Derp. It is actually implemented, except for _getting the isolated view nodes_ with one of the default parse methods.

So yes, _applying isolate nodes_ was implemented yet parsing the currently isolated ones wasn't and still isn't.

Thanks for correcting me! :)

**Comment by mottosso** [comment]
> So yes, applying isolate nodes was implemented yet parsing the currently isolated ones wasn't and still isn't.

Wait wait wait, I thought we said we didn't *need* the parse version of this, as it didn't make sense? From [this](https://github.com/abstractfactory/maya-capture/issues/61#issuecomment-242867737) comment.

**Comment by BigRoy** [comment]
Ah, the comment wasn't intended to describe a _missing feature_ but to indicate the path was eventually chosen as this particular issue was closed. So it described the end result and the reason it was closed (and thus solved).

We've decided that documenting on how to parse it manually would suffice instead of adding it to the readily available parse function as the resulting values (currently isolated nodes in a view) are very specific to a certain state of a scene that can hardly be re-applied in many scenes (node names tend to be different in different scenes). As such it's much less of a re-usable piece of information. As indeed described in that comment.

In short, what we decided on _documentation is in there and implemented_ yet _it's not a parsing method_ as originally described.

**Comment by mottosso** [comment]
Cool!
